pack/systemd_dir.go

```diff
@@ -0,0 +1,223 @@
+package pack
+
+import (
+	"fmt"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+	"github.com/tarantool/cartridge-cli/templates"
+)
+
+type systemdCtx struct {
+	Name       string
+	AppDir     string
+	WorkDir    string
+	Entrypoint string
+
+	TarantoolDir string
+
+	StateboardName       string
+	StateboardWorkDir    string
+	StateboardEntrypoint string
+}
+
+var (
+	systemdAppFilesTemplate = templates.FileTreeTemplate{
+		Dirs: []templates.DirTemplate{
+			{
+				Path: "/etc/systemd/system/",
+				Mode: 0755,
+			},
+		},
+		Files: []templates.FileTemplate{},
+	}
+
+	defaultAppUnitTemplate = templates.FileTemplate{
+		Path:    "/etc/systemd/system/{{ .Name }}.service",
+		Mode:    0644,
+		Content: appUnitContent,
+	}
+
+	defaultAppInstUnitTemplate = templates.FileTemplate{
+		Path:    "/etc/systemd/system/{{ .Name }}@.service",
+		Mode:    0644,
+		Content: appInstUnitContent,
+	}
+
+	defaultStateboardUnitTemplate = templates.FileTemplate{
+		Path:    "/etc/systemd/system/{{ .StateboardName }}.service",
+		Mode:    0644,
+		Content: stateboardUnitContent,
+	}
+)
+
+func initSystemdDir(baseDirPath string, projectCtx *project.ProjectCtx) error {
+	log.Debugf("Create systemd dir in %s", baseDirPath)
+
+	ctx := systemdCtx{
+		Name:       projectCtx.Name,
+		AppDir:     filepath.Join("/usr/share/tarantool", projectCtx.Name),
+		WorkDir:    filepath.Join("/var/lib/tarantool/", projectCtx.Name),
+		Entrypoint: project.AppEntrypointName,
+
+		StateboardName:       projectCtx.StateboardName,
+		StateboardWorkDir:    filepath.Join("/var/lib/tarantool", projectCtx.StateboardName),
+		StateboardEntrypoint: project.StateboardEntrypointName,
+	}
+
+	if projectCtx.TarantoolIsEnterprise {
+		ctx.TarantoolDir = ctx.AppDir
+	} else {
+		ctx.TarantoolDir = "usr/bin" // TODO
```

Comment:
`/usr/bin/env tarantool`

Reply:
Decided to fix it later, now fixed to `/usr/bin`.